### PR TITLE
Fixed major Altair bugs

### PR DESCRIPTION
**Pull request by jluhrs**
Reworked Altair control code. The main difference is that now the AO guide is not stopped when using LGS mode. Also, the dependency on the parameter "field lens" was removed.

### CODE REVIEW

File: modules/server/src/main/scala/seqexec/server/altair/AltairControllerEpics.scala [hunk 1]
```diff
@@ -95,84 +94,142 @@ object AltairControllerEpics {
...
-      (newPosOk && guideOk).option(
-        (L.debug("Resume Altair guiding") *>
-          epicsAltair.waitMatrixCalc(CarStateGEM5.IDLE, MatrixPrepTimeout) *>
-          dmFlattenAction *>
-          epicsTcs.aoCorrect.setCorrections(CorrectionsOn) *>
-          epicsTcs.targetFilter.setShortCircuit(TargetFilterOpen) *>
-          epicsTcs.targetFilter.post(DefaultTimeout) *>
-          L.debug("Altair guiding resumed") *>
+    // Let's keep this check until we are sure the coordinates for the control matrix are properly estimated
+    private val checkControlMatrix: F[Unit] = {
```
Comment (jluhrs) [inline modules/server/src/main/scala/seqexec/server/altair/AltairControllerEpics.scala:195]: I am not sure the position prediction is correct. This warning will tell.

File: app/seqexec-server-gs/src/main/resources/app.conf [hunk 2]
```diff
@@ -61,7 +61,7 @@ seqexec-engine {
   epicsCaAddrList = "172.17.2.255 172.17.3.255 172.17.102.130 172.17.105.20 172.16.102.130 172.17.106.111 172.17.105.37 172.17.107.50 172.17.55.101 172.17.101.101 172.17.65.255 172.17.102.139 172.17.102.138"
   readRetries = 4
   ioTimeout = 4 seconds
-  dhsTimeout = 24 seconds
+  dhsTimeout = 30 seconds
```
Comment (jluhrs) [inline app/seqexec-server-gs/src/main/resources/app.conf:64]: This is an unrelated change. This timeout was increased manually months ago because of some problems with the DHS.

File: .scalafmt.conf [hunk 3]
```diff
@@ -14,8 +14,8 @@ rewrite.rules = [
   PreferCurlyFors
 ]
 
-align.tokens.add = [{code = "=>", owner = "Case"}]
-align.tokens.add = [{code = ":", owner = "Term.Param"}, "=", "shouldBe", "<-", "^"]
+align.tokens."+" = [{code = "=>", owner = "Case"}]
+align.tokens."+" = [{code = ":", owner = "Term.Param"}, "=", "shouldBe", "<-", "^"]
```
Comment (jluhrs) [inline .scalafmt.conf:18]: This change is to avoid a scalafmt complain.

File: app/seqexec-server-gs/src/main/resources/app.conf [hunk 4]
```diff
@@ -48,8 +48,8 @@ seqexec-engine {
       ghostGds = simulated
       gmos = full
       gnirs = simulated
-      gpi = full
-      gpiGds = full
+      gpi = simulated
```
Comment (cquiroz) [inline app/seqexec-server-gs/src/main/resources/app.conf:51]: Are you making this permanent?

File: modules/server/src/main/scala/seqexec/server/altair/AltairControllerEpics.scala [hunk 5]
```diff
@@ -95,84 +94,142 @@ object AltairControllerEpics {
...
 
-      needsToStop.option((newCfg, pause))
     }
 
+    private val setCorrectionsOff: F[ApplyCommandResult] =
+      epicsTcs.aoCorrect.setCorrections(CorrectionsOff) *> epicsTcs.aoCorrect.post(DefaultTimeout)
+
+    private val setCorrectionsOn: F[ApplyCommandResult] =
+      epicsTcs.aoCorrect.setCorrections(CorrectionsOn) *>
+        epicsTcs.aoCorrect.setGains(1) *>
```
Comment (jluhrs) [inline modules/server/src/main/scala/seqexec/server/altair/AltairControllerEpics.scala:150]: Without this, Altair will reset the adaptive gains when restarting the guiding leading to longer settle times.

Comment (cquiroz) [reply modules/server/src/main/scala/seqexec/server/altair/AltairControllerEpics.scala:150]: Maybe it deserves a comment and/or a constant rather than `1`

**Review by cquiroz — APPROVED** [review]
LGTM
